### PR TITLE
fixed layout for edit properties host view

### PR DESCRIPTION
**Pull request by ynnaah**

### CODE REVIEW

File: frontend/src/api-client.ts
Comment (karumadesu) [inline frontend/src/api-client.ts:1]: duplicate function? see line 192

File: frontend/src/components/Header.tsx
Comment (karumadesu) [inline frontend/src/components/Header.tsx:1]: line 16? missing closing angle brackets + closing link tag

**Review by karumadesu — CHANGES_REQUESTED** [review]
see file comments

**Review by ynnaah — COMMENTED** [review]
updated

**Review by karumadesu — APPROVED** [review]

**Pull request merged**